charts/istio-gateway/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for common.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+gatewayClass: "istio-ingressgateway"
+replicaCount: 2
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using Release.Name
+  name: ""
+
+resources:
+  limits:
+    cpu: 2000m
+    memory: 1024Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+
+strategy:
+  rollingUpdate:
+    maxSurge: 25%
+    maxUnavailable: 25%
+  type: RollingUpdate
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  metrics: []
+
+hpaBehavior:
+  enabled: false
+  behavior: {}
+#   scaleDown:
+#     stabilizationWindowSeconds: 300
+#     policies:
+#     - type: Percent
+#       value: 100
+#       periodSeconds: 15
+#   scaleUp:
+#     stabilizationWindowSeconds: 0
+#     policies:
+#     - type: Percent
+#       value: 100
+#       periodSeconds: 15
+#     - type: Pods
+#       value: 4
+#       periodSeconds: 15
+#     selectPolicy: Max
+
+# Type Resource
+#
+#  metrics:
+#  - type: Resource
+#    resource:
+#      name: cpu
+#      target:
+#        type: Utilization
+#        averageUtilization: 50
+#  - type: Resource
+#    resource:
+#      name: memory
+#      target:
+#        type: AverageValue
+#        averageValue: 100Mi
+#
+# Type External
+#
+#  metrics:
+#  - type: External
+#    external:
+#      metric:
+#        name: pubsub.googleapis.com|subscription|num_undelivered_messages
+#        selector:
+#          matchLabels:
+#            resource.labels.subscription_id: payment-method-update
+#      target:
+#        type: AverageValue
+#        averageValue: 100
+
+# PodDisruptionBudget
+#
+# minAvailable and maxUnavailable are mutually exclusive, only one should be set.
+#
+# If podDisruptionBudget is enabled but no minAvailable or maxUnavailable fields
+# are set, the PodDisruptionTemplate template will default to `minAvailable: 15%`.
+#
+# The value for minAvailable and maxUnavailable should either be set to an integer
+# or a percentage.  For more information see the official documentation:
+#
+# https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+podDisruptionBudget:
+  minAvailable: "15%"
```

Comment:
I think this is way too aggressive. maxUnavailable: 1 is what we have been defaulting to. I don't want k8 to start taking down 85% of my capacity because it's trying to do whatever monkey business it does.